linked_list/linked_list.py

```diff
@@ -23,6 +23,13 @@ def __init__(self):
     self.tail = None
 
   def add_to_tail(self, value):
+    if self.head is None:
+      new_node = Node(value)
+      self.head = new_node
+      self.tail = new_node
+    else:
+      self.tail.next_node = Node(value)
+      self.tail = self.tail.next_node
     pass
 
   def remove_head(self):
```

Comment:
Some tips on on how to get delete working. 

- Check if there is a currently a head. Return `None` no head
- save the value of head to a var
- change the value of head to the current heads next_node
- check if the new head is `None`, if it is, you know the list is now empty. set the tail to `None`
- return the value of the old head